src/middlewared/middlewared/plugins/filesystem_/acl_template.py

```diff
@@ -116,6 +132,7 @@ async def do_update(self, id, data):
         return await self.get_instance(id)
 
     @accepts(Int('id'))
+    @returns()
```

Comment:
This is not required and middleware will automatically add this for delete call in CRUD services